src/xhydro_lstm/lstm_functions.py

```diff
@@ -0,0 +1,964 @@
+"""Collection of functions required to process LSTM models and their required data."""
+
+import numpy as np
+import tensorflow as tf
+import tensorflow.keras.backend as k
+from sklearn.preprocessing import MinMaxScaler, StandardScaler
+
+# TODO: Cannot use star imports, pre-commit screams.
```

Comment:
You can, we just need to set that to an allowed check in the `pyproject.toml` (ignore `F405`).